docs/releases/dev.rst

```diff
@@ -34,6 +34,10 @@ Details of changes
   is now built on the client. This requires refreshing all server stats using
   the :djadmin:`refresh_stats_rq` command (:issue:`3835`).
 
+- Disabled projects are visually differentiated in the projects drop-down
+  (:issue:`3996`). Since the in-cache data structure supporting this changed,
+  it's necessary to clear the cache.
```

Comment:
>> Perhaps it is a good idea to provide some hint on how to do that.

> Any specific user-friendly suggestions? if you read the PR description I'm asking for help on that.

No. I can only think on asking the user to open a shell and run some simple Python code. But that is definitely not friendly. On the other hand throwing away tons of calculated stats sounds really really scary. Perhaps we should expand `clear_stats` to allow to remove some specific cache by providing some identifier, like `--projectdropdowns`.